docs/framework/wpf/app-development/build-and-deploy-how-to-topics.md

```diff
@@ -8,22 +8,21 @@ helpviewer_keywords:
   - "WPF application [WPF], deploying"
 ms.assetid: 88952ad2-5b74-48ca-a4c5-3f4fbb53ce12
 ---
-# Build and Deploy How-to Topics
-The following topics show how to create project files for the various [!INCLUDE[TLA2#tla_winclient](../../../../includes/tla2sharptla-winclient-md.md)] application types.  
-  
-## In This Section  
- [Configure IIS 5.0 and IIS 6.0 to Deploy WPF Applications](how-to-configure-iis-5-0-and-iis-6-0-to-deploy-wpf-applications.md)  
- [Configure Visual Studio to Debug a XAML Browser Application to Call a Web Service](configure-vs-to-debug-a-xaml-browser-to-call-a-web-service.md)  
- [Determine the Installed Version of WPF](how-to-determine-the-installed-version-of-wpf.md)  
```

Comment:
Why not change the link to the archive version instead of deleting it?

Reply:
@Thraka, I didn't notice your comment to update them until just now.
I deleted them based on @gewarren comment.

> They should be removed.

I don't have a personal preference for the two options, so you and @gewarren may discuss that with each others. I'm okay with both options.

Reply:
If we leave the link in the TOC and in this file, then we should indicate that it's for 3.5 and below only. But it seems a bit weird to me to link to a previous-versions article from a current TOC.

Reply:
It doesn't have to be in the TOC. I say keep it out of the TOC and just have this link here. It may be helpful for someone. These types of link farm articles are MSDN carryover, so I think it's OK to have something in here that isn't in the TOC.

Reply:
@Thraka, Ok. I've added the link back to the "In this section" heading.

Reply:
It's ok to have links here but "in this section" section should match the TOC. So the previous-versions link should be in the Related sections.